<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerPanel</class>
 <widget class="QMainWindow" name="CustomerPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="mainLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="4" column="0">
     <widget class="QPushButton" name="passwordEditButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/images/panels/password.png</normaloff>:/images/panels/password.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="noAccountLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <fontweight>ExtraLight</fontweight>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Sunken</enum>
      </property>
      <property name="text">
       <string>You have no accounts yet.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <spacer name="spacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::MinimumExpanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="title">
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Welcome back, Amir Mohsen</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="transferButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/images/panels/cash.png</normaloff>:/images/panels/cash.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QListView" name="accountView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLabel" name="description">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <fontweight>Light</fontweight>
       </font>
      </property>
      <property name="text">
       <string>You can view your accounts and transfer some money</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>accountView</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>CustomerPanel</receiver>
   <slot>viewAccount(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>815</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transferButton</sender>
   <signal>clicked()</signal>
   <receiver>CustomerPanel</receiver>
   <slot>transfer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>728</x>
     <y>752</y>
    </hint>
    <hint type="destinationlabel">
     <x>782</x>
     <y>677</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passwordEditButton</sender>
   <signal>clicked()</signal>
   <receiver>CustomerPanel</receiver>
   <slot>changePassword()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>747</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>730</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>viewAccount(QModelIndex)</slot>
  <slot>changePassword()</slot>
  <slot>transfer()</slot>
 </slots>
</ui>
